### DISCUSSION TITLE
Error while deploying new project

### DISCUSSION THREAD

**Discussion by jhcueva** [vercel/vercel#8777] [General] [answered]
When I import a [repository](https://github.com/jhcueva/Pokedex) from my github account, name the project as **Pokedex** and click on deploy I get the following error.
**Error:The name of a Project can only contain up to 100 alphanumeric lowercase characters and hyphens.**
I used Vite as framework preset

**Comment by amyegan** [comment, marked as answer]
Hi @jhcueva! It looks like the name of your repo is not being automatically converted to lowercase for some reason. If you manually change the Project Name to use only lowercase characters, then it will deploy. I'll let the team know about this so we can get it working more smoothly. Thanks for bringing this up!

**Reply by upalatucci** [reply]
Hei guys. I'm having the same issue now. The project has uppercase characters but no issues until now :D
